docs/guides/admin/docs/workflowoperationhandlers/configure-by-dcterm-woh.md

```diff
@@ -15,7 +15,8 @@ Tags and flavors can be used in combination.
 |------------------|-------|-----------|-------------|
 |dccatalog         |"episode" or "series"|the type of catalog in which to search for `dcterm`|EMPTY|
 |dcterm            |"creator"            |the name of the Dublin Core term which to check|EMPTY|
-|match-value       |"Joe Bloggs"         |the Dublin Core term value to check for|EMPTY|
+|externalcatalog   |i.e. "myterms"       |the type of external metadata catalog in which to search for `dcterm`|dublincore|
```

Comment:
Maybe, instead of treating dublincore as a special case, it would make sense to have a more generic configuration of source catalog and source element like the [transfer-metadata operation](https://github.com/opencast/opencast/blob/d1aedd50967a2fc8564e0f6bc3462f5b06a8a370/docs/guides/admin/docs/workflowoperationhandlers/transfer-metadata-woh.md#operation-example)